.github/workflows/validate-plugin.yml

```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Use Node.js 16
+      - name: Use Node.js 18
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Should we add `.nvmrc` file with predefined version of Node.js to be used in the project?

Reply:
No